google/cloud/security/common/data_access/forseti.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Forseti Database Objects. """
+
+from sqlalchemy import create_engine
+from sqlalchemy import Column, String
+from sqlalchemy import Text, BigInteger, Date
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+BASE = declarative_base()
+
+
+# pylint: disable=R0914
```

Comment:
Please also use the common name for the disables rather than the error code. Feel free to also have them share one statement separated by comments.